docs/section22/Rule22-5.md

```diff
@@ -0,0 +1,59 @@
+
+# CHW&CW - Rule 22-5  
+
+**Rule ID:** 22-5  
+**Rule Description:** For Baseline chilled water loop that is not purchased chilled water and serves computer room HVAC systems (System Type-11), chilled-water supply temperature shall be reset higher based on the HVAC system requiring the most cooling.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.9 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system serves computer room HVAC system (System Type-11)
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+3. Pass Rule 22-3.
+
```

Comment:
I don't think #3 belongs here. My understanding of the rules is that if loops serves at least one system 11 its supply temperature should be reset based on load instead of outdoor air temperature.

Reply:
Right I will remove this.